### PR TITLE
[12.0][MIG] stock_request: Migrated.

### PR DESCRIPTION
**Pull request by murtuzasaleh**
**WIP**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@murtuzasaleh Thanks for your contribution. Could you maybe change your PR title to be a little bit clearer? If it is a migration to 12, you can say [12.0][MIG] stock_request

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh Thanks for your contribution. Could you maybe change your PR title to be a little bit clearer? If it is a migration to 12, you can say [12.0][MIG] stock_request

Sure I will do it, now It is WIP.

**Comment by nikul-serpentcs** [comment]
@pedrobaeza Can you please check why Runbot is Rad/False?

**Comment by pedrobaeza** [comment]
```
2019-03-20 11:28:06,596 161 WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (route_ids, route_id) of stock.request.abstract() have the same label: Route.
2019-03-20 11:28:06,609 161 WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (route_ids, route_id) of stock.request() have the same label: Route.
```

**Comment by murtuzasaleh** [comment]
@jbeficent @hveficent Can you please review this PR.

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
cherry-pick my commit 2753108d1cead447440cc6258d1570febcb7a1d7 and rebase.

**Comment by murtuzasaleh** [comment]
> cherry-pick my commit [2753108](https://github.com/OCA/stock-logistics-warehouse/commit/2753108d1cead447440cc6258d1570febcb7a1d7) and rebase.

@hveficent I have added that commit [2753108](https://github.com/OCA/stock-logistics-warehouse/commit/2753108d1cead447440cc6258d1570febcb7a1d7) in my PR when your PR is not merged, Can I revert my code and cherry-pick your commit and rebase it?

File: stock_request/security/stock_request_security.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    <record id="group_stock_request_user" model="res.groups">
+        <field name="name">Stock Request User</field>
+        <field name="implied_ids" eval="[(4, ref('base.group_user'))]"/>
+        <field name="category_id" ref="module_category_stock_request"/>
+    </record>
+
+    <record id="group_stock_request_manager" model="res.groups">
+        <field name="name">Stock Request Manager</field>
+        <field name="users" eval="[(4, ref('base.user_root'))]"/>
```
Comment (HviorForgeFlow) [inline stock_request/security/stock_request_security.xml]: Functional review (first attempt):
I'm only able to access to Stock Request General Settings if I use Become a SuperUser, i think problem is that now admin user is defined as base.user_admin

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
:man_firefighter:

**Comment by max3903** [comment]
@osi-scampbell Please test, review and comment. Thanks!

**Review by osi-scampbell — APPROVED** [review]
Code installs cleanly and everything appears to be working on my end. @murtuzasaleh can you squash you last few commits please?

**Review by max3903 — APPROVED** [review]

**Pull request merged**